### ISSUE TITLE
Blue accent color in dark theme does not meet accessibility standards

### ISSUE THREAD

**Issue by eisenwave** [cppstat/cppstat#94]
<img width="203" height="190" alt="Image" src="https://github.com/user-attachments/assets/0535f2a0-6c9c-42ed-b72f-b36f7bf44dc2" />

The contrast for link colors is really low (3.15:1).

The minimum contrast ratio to meet AA WCAG requirements is 4.5:1, and 7:1 for AAA requirements.
